### PR TITLE
chore: add dependency in apisix-base

### PR DESCRIPTION
**Pull request by Xunzhuo**
see https://github.com/apache/apisix-docker/pull/242

### CODE REVIEW

File: .github/workflows/push-apisix-base-image.yml [hunk 1]
```diff
@@ -39,7 +39,16 @@ jobs:
         run: |
           docker build -t api7/apisix-base:${{ steps.tag_env.outputs.version }} \
             -f ./dockerfiles/Dockerfile.apisix-base.apk .
-      
+
+      - name: Run apisix-base docker
```
Comment (spacewander) [inline .github/workflows/push-apisix-base-image.yml:43]: Better to verify it in the build but not the upload. Please add it to https://github.com/api7/apisix-build-tools/blob/e3a5cc9abf66a82892b61582038e03a53d2d323d/.github/workflows/package-apisix-base-apk.yml#L30

Comment (Xunzhuo) [reply .github/workflows/push-apisix-base-image.yml:43]: OK Thanks~

**Comment by Xunzhuo** [comment]
@spacewander Need to check again, if it is OK, please merge it thanks. After merging this PR, I can test CI: https://github.com/api7/apisix-build-tools/pull/130

**Review by spacewander — COMMENTED** [review]
Could we merge https://github.com/api7/apisix-build-tools/pull/130 into this one? There is no good reason to split them.

**Comment by Xunzhuo** [comment]
I thought #130 is a CI task and this PR is a dependency task, so I separate them. But it is OK to merge #130 together.

**Review by spacewander — APPROVED** [review]

**Pull request merged**